.github/workflows/r_artifacts.yml

```diff
@@ -0,0 +1,34 @@
+name: R artifact builds
+
+on:
+  pull_request_review_comment:
+    types: [created]
+
+jobs:
+  cran-package:
+    name: cran-package
+    if: github.event.comment.body == '/gha build r-artifacts' && contains('OWNER,MEMBER,COLLABORATOR', github.event.comment.author_association)
+    timeout-minutes: 120
```

Comment:
Is this a copy-paste artifact from `r_valgrind.yml` or we really need more than our standard 1hr?

Reply:
copy-paste from r_valgrind, let's remove it

```suggestion
```

Reply:
I think we need our standard 60mins for this build.
https://github.com/microsoft/LightGBM/blob/03c4d45541166b6fdaccd4aef9b126a0084758ed/.github/workflows/r_package.yml#L15

I remember you've suggested them to replace default 360mins and this sounds very reasonable.

Reply:
ok yes you're right, I'll set it to that.

Reply:
just added: https://github.com/microsoft/LightGBM/pull/3497/commits/4553027b6a730f5b5cb1e74ccd60d9a5774ba55e